pkg/network/ebpf/c/task_event.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __TASK_EVENT_H
+#define __TASK_EVENT_H
+
+#define RETURN_IF_NOT_IN_SYSPROBE_TASK(prog_name)           \
```

Comment:
You can #define this to nothing if we're not on fentry. You will have to add a preprocessor define on the command line for this likely, something like `USE_FENTRY=1` and check for it here. This would only be defined when compiling `tracer-fentry.c`. The code to compile is here: https://github.com/datadog/datadog-agent/blob/main/tasks/system_probe.py#L229